vmcnet/utils/io.py

```diff
@@ -155,6 +158,13 @@ def reload_vmc_state(directory: str, name: str) -> CheckpointData:
             return (epoch, data, params, optimizer_state, key)
 
 
+def reload_model_state(path: str):
```

Comment:
Instead of creating a new method here why not just use `reload_vmc_state(directory, name)[2]` to get the params?